packages/expo-cli/src/commands/build/ios/IOSBuilder.ts

```diff
@@ -322,6 +327,26 @@ See https://docs.expo.io/versions/latest/distribution/building-standalone-apps/#
       }
     }
   }
+
+  // warns for "damaged" builds when targeting simulator
+  // see: https://github.com/expo/expo-cli/issues/1197
+  maybeWarnDamagedSimulator() {
+    // see: https://en.wikipedia.org/wiki/Darwin_%28operating_system%29#Release_history
+    const isMacOsCatalinaOrLater =
+      os.platform() === 'darwin' && semver.satisfies(os.release(), '>=19.0.0');
+
+    if (isMacOsCatalinaOrLater && this.options.type === 'simulator') {
+      log.newLine();
+      log(
+        chalk.bold(
+          `🚨 If the build is not installable on your simulator because of "${chalk.underline(
+            `... is damaged and can't be opened.`
+          )}", please run:`
+        )
+      );
+      log(chalk.grey.bold('xattr -rd com.apple.quarantine /path/to/your.app'));
```

Comment:
we should provide a full set of instructions on how to run this, perhaps on the expo.fyi page

eg: we should tell people they need to extract the tar.gz and then drag into simulator